sdk/iot/Azure.IoT.Hub.Service/api/Azure.IoT.Hub.Service.netstandard2.0.cs

```diff
@@ -525,9 +525,26 @@ public partial class DeviceRegistryOperationWarning
     {
         internal DeviceRegistryOperationWarning() { }
         public string DeviceId { get { throw null; } }
-        public string WarningCode { get { throw null; } }
+        public Azure.IoT.Hub.Service.Models.DeviceRegistryOperationWarningCode? WarningCode { get { throw null; } }
```

Comment:
https://apiview.dev/Assemblies/Review/944e91042ebe4e16a60ce6175882dde1/3c561c389e75457cb717cdebac0c8488?diffRevisionId=c48b3f26e18845e6b02203d0b3060e1c&doc=False&diffOnly=True#bc7487eeecd644f6af3b2a53d341cc02

Reply:
I only added the EBNever attributes when ApiCompat reports a failure. This package is still in preview: https://github.com/Azure/azure-sdk-for-net/blob/20aaff726a83733f6ce5cf720c5cc249c641e0ae/sdk/iot/Azure.IoT.Hub.Service/src/Azure.IoT.Hub.Service.csproj#L13 do we need to add EBNever to this? Because we have to come up with a new name